### PR TITLE
Fix tinymce format icon names

### PR DESCRIPTION
**Pull request by petschki**
upgradestep for https://github.com/plone/plone.base/pull/57

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: plone/app/upgrade/v60/final.py [hunk 1]
```diff
@@ -206,3 +206,23 @@ def fix_syndication_settings(context):
...
+
+    See https://github.com/plone/Products.CMFPlone/issues/3905
+    """
+    registry = getUtility(IRegistry)
+    record = registry.records.get("plone.inline_styles")
+    if record is None:
+        return
+    values = record.value
+    replacements = [
+        ("Superscript|superscript|superscript", "Superscript|superscript|super-script"),
```
Comment (mauritsvanrees) [inline plone/app/upgrade/v60/final.py:222]: This is fixing the wrong line.  Afterwards, the control panel has this:

```
Strikethrough|strikethrough|strikethrough
Superscript|superscript|super-script
```

Comment (petschki) [reply plone/app/upgrade/v60/final.py:222]: Ouch sorry. Will fix it asap.

Comment (petschki) [reply plone/app/upgrade/v60/final.py]: fixed

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
Thanks!

**Pull request merged**